search/releases/7.0/index.adoc

```diff
@@ -0,0 +1,66 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: search
+:awestruct-series_version: "7.0"
+:page-interpolate: true
+:hsearch-doc-url-prefix: #{reference_doc(site.projects[page.project], series(page, page.series_version)).html_url}
+
+=== Dependency upgrades
+
+[[java-version]]
+Java::
+Hibernate Search is now using JDK 11 as a baseline and can work on JDK 11, 17, 20, 21 or 22.
```

Comment:
```suggestion
Hibernate Search is now using JDK 11 as a baseline and can work on JDK 11, 17 or 21.
Compatibility with JDK 8 was dropped.
```

(we probably should be explicit about that last part, as it's the one with most consequences for users)